torch/_C/__init__.pyi.in

```diff
@@ -854,6 +854,7 @@ class FunctionSchema:
     returns: List[Argument]
     name: str
     overload_name: str
+    is_mutable: _bool
```

Comment:
Why do we have this field and not just look it up on the arguments?

Reply:
Isn't that what you do in your function is_functional that you added below?

Reply:
Is your suggestion to delete `is_mutable` and add a helper function `is_mutable` to do what it previously did? If so, there already is an `is_mutable` helper function in C++ (that's what this field is).

NB: `is_mutable` and `is_functional_schema` below aren't opposites :/ we have too many variants of these checks flying around and I'm trying to not break any existing code (is_mutable exists already today).

Reply:
NB: is_mutable is a @property, so we're good.